src/App.tsx

```diff
@@ -160,6 +161,13 @@ const AppRoutes = ({ base }: AppProps) => {
                     <Route path=":tag" element={<TagsPage />} />
                   </Route>
                 </Route>
+                <Route path="folder/:folderId">
+                  <Route index element={<SharedFolderPage />} />
+                  <Route
+                    path=":subfolderId/:resourceId"
```

Comment:
Vil dette sei at du kun kan vise ett undernivå?

Reply:
Nei. Dette er satt opp på samme måter som mapper i Min NDLA. `subFolderId` kan referere til en mappe som ligger flere nivåer dypt. Er bare en løsning for å få kortere URL'er.